examples/foam/index.html

```diff
@@ -2,11 +2,10 @@
 <head>
   <meta charset="utf-8">
   <title>FOAM • TodoMVC</title>
-  <link rel="stylesheet" href="bower_components/todomvc-common/base.css">
-  <link rel="stylesheet" href="bower_components/todomvc-app-css/index.css">
-  <script src="bower_components/foam/build/foam.js"></script>
-  <script src="bower_components/todomvc-common/base.js"></script>
-  <script src="Todo.js"></script>
+  <link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+  <link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+  <script language="javascript" src="node_modules/todomvc-common/base.js"></script>
```

Comment:
Yeah, that's an archaism. Even with an uncooperative server that didn't supply the right headers, it should probably be spelled type="application/javascript" now or something. I've removed it.